scipy/stats/tests/test_distributions.py

```diff
@@ -2381,6 +2381,34 @@ def test_logcdf(self):
                         stats.norm.logsf(np.log(x2-mu)/sigma))
 
 
+# Columns are x, a, b, cdf, sf.
+# The values of cdf and sf were computed using mpmath.
+_beta_cdf_sf = np.array([
+    [1e-20, 3.0, 0.5, 3.1249999999999994e-61, 1.0],
+    [1e-20, 0.5, 3.0, 1.875e-10, 0.9999999998125],
+    [0.999, 2, 10, 1.0, 1.0990000000000097e-29],
```

Comment:
This is an improvement. Currently, beta.sf(0.999, 2, 10) returns `1.1102230246251565e-16`.

Reply:
So along the lines of the comment below, the `sf` should not have the `97` at the end.